services/app-api/serverless.yml

```diff
@@ -128,12 +127,12 @@ provider:
     SECRETS_MANAGER_SECRET: ${self:custom.secretsManagerSecret}
     EMAILER_MODE: ${self:custom.emailerMode}
     PARAMETER_STORE_MODE: ${self:custom.parameterStoreMode}
-    SES_SOURCE_EMAIL_ADDRESS: ${self:custom.emailSource}
-    SES_REVIEW_TEAM_EMAIL_ADDRESSES: ${self:custom.emailReviewTeamAddresses}
-    SES_REVIEW_HELP_EMAIL_ADDRESS: ${self:custom.emailReviewHelpAddress}
-    SES_RATE_HELP_EMAIL_ADDRESS: ${self:custom.emailRateHelpAddress}
-    SES_DEV_TEAM_HELP_EMAIL_ADDRESS: ${self:custom.emailDevTeamHelpAddress}
-    SES_RATES_EMAIL_ADDRESSES: ${self:custom.emailRatesTeamAddresses}
+    SES_SOURCE_EMAIL_ADDRESS: ${ssm:/configuration/email/sourceAddress}
```

Comment:
I’m glad we are still configuring the app using env vars, constraining the SSM stuff to serverless.yml is great.